modules/post-installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-configuring-nutanix-failure-domains_{context}"]
+= Adding failure domains to the Infrastructure CR
+
+You add failure domains to an existing Nutanix cluster by modifying its Infrastructure custom resource (CR) (`infrastructures.config.openshift.io`).
+
+[TIP]
+====
+It is recommended that three failure domains be configured to ensure high-availability.
+====
+
+.Procedure
+
+. Edit the Infrastructure CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit infrastructures.config.openshift.io cluster
+----
+
+. Configure the failure domains.
++
+.Example Infrastructure CR with Nutanix failure domains
```

Comment:
```suggestion
.Example Infrastructure CRD with Nutanix failure domains
```

Reply:
This was raised early as part of an SME review. The decision was to go with CR.

https://github.com/openshift/openshift-docs/pull/69618#discussion_r1457598396